pyttb/sptensor.py

```diff
@@ -688,16 +924,43 @@ def isequal(self, other: Union[sptensor, ttb.tensor]) -> bool:
 
     def logical_and(self, B: Union[float, sptensor, ttb.tensor]) -> sptensor:
         """
-        Logical and with self and another object
+        Logical AND with a scalar or another sparse or dense tensor.
```

Comment:
Numpys logical and is a function on two objects but their documentation takes a similar approach of not being too descriptive in the raw text, but instead the parameter annotations are more verbose.
```suggestion
        Logical AND with another object.
```